photoprism/docker-compose.yaml

```diff
@@ -33,18 +33,18 @@ services:
     stop_grace_period: 10s
     depends_on:
       - mariadb
-# are these security options safe? or do i want `no-new-privileges:true` and `cap_drop:      - ALL`?
     security_opt:
-      - seccomp:unconfined 
-      - apparmor:unconfined
+      - no-new-privileges:true
+      #- seccomp:unconfined 
+      #- apparmor:unconfined
     #ports:
     #  - "2342:2342" # HTTP port (host:container)
     environment:
       - PHOTOPRISM_ADMIN_USER
       - PHOTOPRISM_ADMIN_PASSWORD
       - PHOTOPRISM_AUTH_MODE
-      - PHOTOPRISM_SITE_URL
-      - PHOTOPRISM_DISABLE_TLS
+      - PHOTOPRISM_SITE_URL=https://${PHOTOPRISM_TRAEFIK_HOST}
+      - PHOTOPRISM_DISABLE_TLS=false
```

Comment:
I think DISABLE_TLS should be true. Because traefik will be adding the TLS on top.

Reply:
Fixed.